example/generator/GeneratorMetadataProvider.js

```diff
@@ -109,6 +118,100 @@ define([
                     }
                 }
             ]
+        },
+        'example.spectral-generator': {
```

Comment:
Why is this stuff showing up? I think maybe `form-refactor` has changes from `master` that `topic-form-refactor` does not?